ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -52,15 +52,25 @@
   retries: 12
   delay: 5
 
-- name: create '_users' database for singleton mode
+- name: check if '_users' database exists
```

Comment:
@dgrove-oss with a persisted store this db may already have been created and hence will fail if tried again.